fetch_bringup/launch/include/aggregator.launch.xml

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <node 
+        pkg="diagnostic_aggregator" type="aggregator_node"
+        name="diag_agg" args="CPP" output="screen" >
```

Comment:
while it is longer, I would suggest making the name "diagnostic_aggregator" to match other robots (like turtlebot)